sympy/physics/mechanics/featherstone.py

```diff
@@ -0,0 +1,78 @@
+# This file will serve as a view of how featherstone method will look if the
+# featherstone class init initialized a system model and the equations of motion
+# are found using a forward_dynamics method just like is done in
+# http://cctbx.sourceforge.net/scitbx_rigid_body_essence/featherstone.py
+
+# NOTE: THIS CODE WILL BE MAKING USE OF THE JOINTS CODE THAT STILL NEEDS TO BE
+# FIXED AND ADDED
+
+
+class Featherstone(object):
+    """Class docstring
+    Discuss the class itself and method
+    Attributes
+    ==========
+
+    Examples
+    ========
+    Double pendulum example
+
+        >>> from sympy import symbols
+        >>> from sympy.physics.mechanics import Body, FeatherstonesMethod,
+        ...     inertia, PinJoint, Point, ReferenceFrame
+
+        >>> mA, mB = symbols('mA mB')
+        >>> mass_center_A = Point('moA')
+        >>> mass_center_B = Point('moB')
+        >>> ground_point = Point('O')
+
+        >>> frameA = ReferenceFrame('A')
+        >>> frameB = ReferenceFrame('B')
+        >>> ground = ReferenceFrame('G')
+
+        >>> body_A_inertia, body_B_inertia = symbols('I_Az I_Bz')
+        >>> body_A_inertia_dyadic = inertia(frameA, 0, 0, body_A_inertia)
+        >>> body_B_inertia_dyadic = inertia(frameB, 0, 0, body_B_inertia)
+        >>> ground_inertia_dyadic = inertia(ground, 0, 0, 0)
+
+        >>> bodyA = Body('A', mass_center_A, mA, frameA, body_A_inertia_dyadic)
+        >>> bodyB = Body('B', mass_center_B, mB, frameB, body_B_inertia_dyadic)
+        >>> ground_body = Body('G', ground_point, 0, ground,
+        ...                    ground_inertia_dyadic)
```

Comment:
There's some lack of clarity in `Body`'s docstring as to when it would fill in information for a body vs. a particle. Also there's an if statement that appears twice in its `__init__()` method where code is being run that wouldn't be used when the second if statement is run. I'm going to put together a PR to alter the `Body` class and link it here.